### PR TITLE
parallellellellellellize patching

### PR DESCRIPTION
**Pull request by tlangs**
### Description

_Please explain the changes you made here._
_Explain the **motivation** for making this change. What existing problem does the pull request solve?_
_Mention any JIRA issues fixed, addressed or otherwise related to this pull request, including issue numbers or hard links for issues in other repos._

----

### Checklist (never delete this)

Never delete this, it is our record that procedure was followed. If you find that for whatever reason one of the checklist points doesn't apply to your PR, you can leave it unchecked but please add an explanation below.

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: Ensure that you have added or updated tests
- [ ] **Submitter**: If you're adding/switching libraries or otherwise changing the design, update the [design documentation](https://broadinstitute.atlassian.net/wiki/pages/viewpage.action?pageId=114531509).
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
* Review cycle:
  * Team may comment on PR at will
  * **Reviewer assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to reviewer** for further feedback
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Move the JIRA issue to QA once this checklist is completed

### CODE REVIEW

File: clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/PatchExecutor.scala [hunk 1]
```diff
@@ -20,6 +20,8 @@ class PatchExecutor[CI <: ClioIndex](patchCommand: PatchCommand[CI]) extends Exe
 
   private lazy val queryAllJson: Json = Json.obj()
 
+  private lazy val parallelism = 32
```
Comment (tbl3rd) [inline clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/PatchExecutor.scala]: Does this help?
Why 32?
How do parallel requests to the server keep the client from timing out?
More smaller upserts?

**Review by tbl3rd — APPROVED** [review]

File: clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/PatchExecutor.scala (hunk 1 above)
Comment (tlangs) [reply clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/PatchExecutor.scala]: It means the client isn't waiting around as long while the server thinks. It succeeded when run against prod.

Comment (danxmoran) [reply clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/PatchExecutor.scala]: Should put a scaladoc explaining that it's pretty much an arbitrary choice.

**Review by danxmoran — APPROVED** [review]

File: clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/PatchExecutor.scala (hunk 1 above)
Comment (tbl3rd) [reply clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/PatchExecutor.scala]: OK. It works if it works. The comment could include the reason too.

**Pull request merged**